cfme/services/service_catalogs/__init__.py

```diff
@@ -98,6 +101,9 @@ def fill(self, fill_data):
         self.after_fill(was_change)
         return was_change
 
+    submit = Button("Submit")
```

Comment:
**Required** I would expect these to be defined at L66 with the rest of the widgets on this page.

Reply:
removed those buttons. not required in this page for now